app/models/concerns/control_objective_items/finding_pdf_data.rb

```diff
@@ -1,34 +1,34 @@
 module ControlObjectiveItems::FindingPdfData
   extend ActiveSupport::Concern
 
-  def finding_pdf_data finding, hide: [], show: []
+  def finding_pdf_data finding, hide: [], show: [], change_name: []
```

Comment:
```suggestion
  def finding_pdf_data finding, hide: [], show: [], custom_labesl: {}
```
Lo haría un hash, entonces después ponés algo como:

```ruby
custom_labels['title'] || finding.class.human_attribute_name('title')
```

Directamente dentro del interpolado, y le mandás la etiqueta desde cualquier formato.